dev/com.ibm.websphere.appserver.features/visibility/public/mpJwt-2.1/io.openliberty.mpJwt-2.1.feature

```diff
@@ -0,0 +1,27 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpJwt-2.1
+visibility=public
+singleton=true
+IBM-ShortName: mpJwt-2.1
+IBM-API-Package: \
+  org.eclipse.microprofile.jwt; type="stable", \
+  org.eclipse.microprofile.auth; type="stable"
+Subsystem-Name: MicroProfile JSON Web Token 2.1
+-features=com.ibm.websphere.appserver.jwt-1.0, \
+  io.openliberty.appSecurity-4.0, \
```

Comment:
Should this start appSecurity-4.0 or appSecurity-5.0?

Reply:
Agreed. Newer commit uses appSecurity-5.0